projects/plugins/protect/src/js/hooks/use-protect-context/index.js

```diff
@@ -0,0 +1,56 @@
+/* global jetpackProtectInitialState */
+
+/**
+ * Merges the list of installed extensions with the list of extensions that were checked for known vulnerabilities and return a normalized list to be used in the UI
+ *
+ * @param {object} installed - The list of installed extensions, where each attribute key is the extension slug.
+ * @param {object} checked   - The list of checked extensions.
+ * @returns {Array} Normalized list of extensions.
+ */
+function mergeInstalledAndCheckedLists( installed, checked ) {
+	const newList = [];
+	for ( const slug in installed ) {
+		if ( checked.hasOwnProperty( slug ) ) {
+			// camelCase fixedIn.
```

Comment:
Is it something that we need to adjust because of the response body shape?

Reply:
Yes.

On one side, we have the list of plugins currently installed, with name, description and everything.

On the other side, we have the response from the server, with the list of plugins that were checked. In this list we only have the plugin slug.

So there might be a plugin that was recently installed and, therefore, was not checked yet. It must be in the list.

And for the plugins that were checked, we need to fetch its name from the list of installed plugins.

Reply:
Nice, thanks. I also mean specifically to the camel case issue.

Reply:
Yes, the data comes from the server in snake-case... This is something we will most likely have to adjust once the real API is developed so I figured it's ok to do that for now. What we need to make consistent is the names and the data structure from this hook to our components.